app/build.gradle

```diff
@@ -53,4 +53,7 @@ dependencies {
     implementation "androidx.compose.material:material:$versions.compose_material_version"
     implementation "androidx.compose.ui:ui-tooling-preview:$versions.compose_ui_version"
 
+    implementation 'com.squareup.retrofit2:retrofit:2.9.0'
```

Comment:
Versions can go into the project gradle.

Reply:
Yes, it is best practice to keep the version in a separate file, and that's why I put compose version in project.gradle. But as this is only a basic setup and for beginners, I am not paying attention to scalability and modularity for time's sake. But will come back later on these topics one by one.

Reply:
great